src/main/main.cpp

```diff
@@ -36,12 +38,18 @@ auto main(int argc, char *argv[]) -> int {
   jucc::InputParser input_parser = jucc::InputParser(argc, argv);
   std::string file_grammar = input_parser.GetArgument("-g");
   if (file_grammar.empty()) {
-    std::cout << "jucc: usage: jucc -g <grammar_file> -f <input_file>\n";
+    std::cout << "jucc: usage: jucc -g <grammar_file> -f <input_file> -o <output_json_file>\n";
     return 0;
   }
   std::string file_input = input_parser.GetArgument("-f");
   if (file_input.empty()) {
-    std::cout << "jucc: usage: jucc -g <grammar_file> -f <input_file>\n";
+    std::cout << "jucc: usage: jucc -g <grammar_file> -f <input_file> -o <output_json_file>\n";
+    return 0;
+  }
+
+  std::string output_file = input_parser.GetArgument("-o");
+  if (output_file.empty()) {
+    std::cout << "jucc: usage: jucc -g <grammar_file> -f <input_file> -o <output_json_file>\n";
```

Comment:
It's better to merge these three condition in a single if block.
Code duplication.